packages/providers/src.ts/fallback-provider.ts

```diff
@@ -497,7 +489,7 @@ export class FallbackProvider extends BaseProvider {
         // We need to make sure we are in sync with our backends, so we need
         // to know this before we can make a lot of calls
         if (this._highestBlockNumber === -1 && method !== "getBlockNumber") {
-            await this.getBlockNumber();
+            // await this.getBlockNumber();
         }
```

Comment:
This is performed before we execute **any** call through the fallback provider. The reason why we have it is to `sync` with the network with the latest block number. I think that once we get to implement the fallback provider fully we will remove this method altogether